README.md

```diff
@@ -1,2 +1,52 @@
-# design-cfps
-Repo to store Cilium CFP design docs
+This directory contains CFP design proposals for features impacting repos across the
+Cilium Github organization.
+
+# Purpose of CFPs
+
+The purpose of a Cilium Feature Proposal (CFP) is to allow community members to gain feedback
+on their designs from the Committers before the community member commits to
+executing on the design. By going through the design process, developers gain a
+high level of confidence that their designs are viable and will be
+accepted.
+
+NOTE: This process is not mandatory. Anyone can execute on their own design
+without going through this process and submit code to the respective repos.
+However, depending on the complexity of the design and how experienced the
+developer is within the community, they could greatly benefit from going through
+this design process first. The risk of not getting a design proposal approved
+is that a developer may not arrive at a viable architecture that the community will
+accept.
+
+# How to create CFPs
+
+To create a CFP, it is recommended to use the `CFP-001-template.md`
+file as an outline. The structure of this template is meant to provide a starting
+point for people. Feel free to edit and modify your outline to best fit your
+needs when creating a proposal. The title should be `CFP-###-subject.md` 
+where the number is the PR number.
```

Comment:
We tried this in Gateway (I think?), and the flow is not great because you have to open the PR, then rename the file after you've opened it. I'd recommend that the flow be either:
- open a CFP issue in cilium/cilium and use that number here,
- or open an issue here and use that number.

Depending on where we want to track delivery of CFPs.

Also, having an issue allows use to merge CFPs as _provisional_, _implementable_ or some other non-Done state, and then closing the CFP issue (wherever it lives) happens when the CFP is implemented.

Reply:
We currently track CFPs in the respective repos that they affect ([see template](https://github.com/cilium/cilium/issues/new?assignees=&labels=kind%2Ffeature&template=feature_template.md&title=CFP%3A+)). While we could theoretically have a number collision if two CFPs in different repo (cilium and hubble for example) I think the risk is low.

We should probably update that template to include this new repo too

Reply:
I've added that here https://github.com/cilium/cilium/pull/23841